projects/services-ogc/src/lib/owc/owc-json.service.ts

```diff
@@ -121,15 +121,67 @@ export class OwcJsonService {
     return context.features;
   }
 
+  /**
+   * Get Resources whith Folder property but not including Layer-Filtertypes
+   */
+  getGroupResources(context: IOwsContext): IOwsResource[] {
+    const resources = context.features;
+    return resources.filter(r => {
+      const groupName = this.getLayerGroupFromFolder(r);
+      return groupName && !Object.keys(Filtertypes).includes(groupName);
+    });
+  }
+
+  /**
+   * Get Resources without Folder property or Folder is only Layer-Filtertypes
+   */
+  getSingleResources(context: IOwsContext): IOwsResource[] {
+    const resources = context.features;
+    return resources.filter(r => {
+      const groupName = this.getLayerGroupFromFolder(r);
+      return !groupName || Object.keys(Filtertypes).includes(groupName);
+    });
+  }
+
   /** Resource --------------------------------------------------- */
   getResourceTitle(resource: IOwsResource): string {
     return resource.properties.title;
   }
 
-  getLayerGroup(resource: IOwsResource): string {
+  /**
```

Comment:
Here `getLayerGroup()` was replaced with `getResourceFolder()` what the function is really doing.
To get the Title for a LayerGroup and check if the folder should get in a LayerGroup ther is the private function `getLayerGroupFromFolder()`